bukuserver/templates/bukuserver/statistic.html

```diff
@@ -4,48 +4,63 @@
 {% block head %}
   {{ super() }}
   {{ buku.close_if_popup() }}
+  <script>// realtime redrawing "Data created" datetime
+    const UNITS = {day: 60*60*24, hour: 60*60, minute: 60, second: 1};
+    addEventListener('load', function recalcReltime() {
+      let diff = Date.now()/1000 - created.getAttribute('data-timestamp');
+      let unit = (diff < 5 ? "" : Object.keys(UNITS).find(k => diff >= UNITS[k]));
+      created.innerText = (!unit ? "just now" : `${parseInt(diff/UNITS[unit])} ${unit}s ago`);
+      setTimeout(recalcReltime, 1000 * {second: 1, minute: 15, hour: 60, day: 15*60}[unit||'second']);
+    });
+  </script>
```

Comment:
Live recalculation of displayed relative time (with appropriate units)
![data created](https://github.com/user-attachments/assets/b126f6aa-8c78-4d75-a99f-c015605fe9cd)
I'd say this is easier to wrap the mind around than "very many seconds" you'd see otherwise :sweat_smile:

(Refresh rate is: every second when showing time in seconds, four times a minute when showing minutes, once a minute when showing hours, and every quarter of hour when showing days. The reason to have steps shorter than the unit is to reduce discrepancy between displayed and actual time, since the timer is not reset when reopening/refreshing the page.)